app/src/components/BackLink.tsx

```diff
@@ -1,14 +1,14 @@
 import { ReactElement } from 'react'
 
-import ButtonLink from '@components/ButtonLink'
+import StyledLink from '@components/StyledLink'
 
-export interface BackLinkProps {
+export type BackLinkProps = {
   href: string
 }
 
 export const BackLink = (props: BackLinkProps): ReactElement => {
   const { href } = props
-  return <ButtonLink labelKey="backlinkText" href={href} style="unstyled" />
+  return <StyledLink textKey="backlinkText" href={href} external={false} />
```

Comment:
Notes for reviewers: I originally thought the back link should be an unstyled button, but after reviewing the [USWDS guidance for the button component](https://designsystem.digital.gov/components/button/), I decided we should use a regular `<a>` instead.

>  When to consider something else
> Linking between a site’s pages. Use regular links instead.